src/ocm/components/clusterConfiguration/CpuArchitectureDropdown.tsx

```diff
@@ -80,12 +82,18 @@ const CpuArchitectureDropdown = ({
   );
 
   const enabledItems = React.useMemo(() => {
-    return cpuArchitectures.map((cpuArch) => {
+    return getAllCpuArchitectures().map((cpuArch) => {
+      const disabledReason = !cpuArchitectures.includes(cpuArch)
+        ? getDisabledReasonForCpuArch(architectureData[cpuArch].label)
```

Comment:
I think the above problem is that this message is not accurate, because it always states that it's the version that does not support the CPU arch.

But in reality, there can be 2 different reasons, either the version support, or that the user does not have the "multi" capability.

For those users, no version would appear to support those CPU architectures, even when that's not the case (Eg. 4.13)